### PR TITLE
Add support for Duration/CqlDuration for REST API, SGv2

### LINKED ISSUE #1696: Stargate V2 REST Service should support "Duration" type
(offshoot of #1693)

Looks like Stargate V2 rewritten REST API does not currently support Duration CQL type; it should.

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Fixes a gap in SGv2/REST wherein `Duration` type (`CqlDuration`) not supported as input or output.
Continuation of #1693.

NOTE: contains changes from #1693 (newly added REST integration test for Duration values).

**Which issue(s) this PR fixes**:
Fixes #1696

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: sgv2-restapi/src/test/java/io/stargate/sgv2/restsvc/grpc/ToProtoConverterTest.java [hunk 1]
```diff
@@ -24,21 +25,22 @@
     return new Arguments[] {
       arguments(123, basicType(TypeSpec.Basic.INT), Values.of(123)),
       arguments(-4567L, basicType(TypeSpec.Basic.BIGINT), Values.of(-4567L)),
-      arguments("abc", basicType(TypeSpec.Basic.TEXT), Values.of("abc")),
+      arguments("abc", basicType(TypeSpec.Basic.VARCHAR), Values.of("abc")),
```
Comment (tatu-at-datastax) [inline sgv2-restapi/src/test/java/io/stargate/sgv2/restsvc/grpc/ToProtoConverterTest.java:28]: Unrelated change, to remove warnings: `Basic.TEXT` deprecated should (have) use(d) `Basic.VARCHAR` already

**Review by jeffreyscarpenter — APPROVED** [review]
